shared/clinical-code-sets/medications/belimumab/1/belimumab.snomed.txt

```diff
@@ -0,0 +1,14 @@
+19482111000001102	Belimumab 400mg powder for solution for infusion vials
```

Comment:
Also:
11116001000001106 | BELIMUMAB 120mg powder for conc for solution for injection
11116201000001100 | BELIMUMAB 400mg powder for conc for solution for injection
11115901000001102 | BENLYSTA 120mg powder for concentrate for soln for infusion
11116101000001107 | BENLYSTA 400mg powder for concentrate for soln for infusion